peer/interfaces.go

```diff
@@ -127,3 +129,15 @@ type ChainArbitrator interface {
 	// ForceCloseContract attempts to force close the channel.
 	ForceCloseContract(wire.OutPoint) (*wire.MsgTx, error)
 }
+
+// Sphinx is an interface that abstracts the decryption of onion blobs.
+type Sphinx interface {
+	// DecodeHopIterators batch decodes HTLC onion blobs.
+	DecodeHopIterators([]byte, []hop.DecodeHopIteratorRequest) (
```

Comment:
I think it can be worth adding the args back in, since it can make the code more self documenting. For example the first `[]byte` arg's purpose could be more obvious just from looking at the interface definition.

Reply:
We usually don't add args to interfaces, also this isn't even used in this package and only to pass-through the peer to create the link

Reply:
Was using htlcswitch `ChannelLink` as my point of reference, but I see that a lot of interfaces have args, will change